engine/src/main/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStore.scala

```diff
@@ -122,7 +124,10 @@ case class SqlWorkflowStore(sqlDatabase: WorkflowStoreSqlDatabase) extends Workf
       case (WorkflowStoreState.Running, true) => RestartableRunning.validNel
         // It's in Aborting state and is being restarted
       case (WorkflowStoreState.Aborting, true) => RestartableAborting.validNel
-      case _ => s"Workflow in state $workflowState, and restarted = $restarted cannot be started and should not have been fetched.".invalidNel
+      case _ =>
+        "Workflow %s in state %s and restarted = %b cannot be started and should not have been fetched."
+          .format(workflowStoreEntry.workflowExecutionUuid, workflowState, restarted)
```

Comment:
why this change?

Reply:
I keep seeing this in travis. Without the workflow id one has zero clue what workflow has the problem

Reply:
Oh I meant the change from interpolating to formatting, the workflow ID context I get and appreciate. 😄

Reply:
Hah! I just find it easier to read lines that don't scroll on infinitely. I still don't think we have an agreed upon exact cutoff yet <sup>*cough* scalafmt someday...</sup>